build.gradle

```diff
@@ -4,15 +4,18 @@ buildscript {
 	}
 	repositories {
 		mavenCentral()
```

Comment:
you can probably remove `mavenCentral()`, as `jcenter()` mirrors through to Maven Central.

Reply:
Nice 👍 I'm picking up so much here :)

Reply:
Oh, good point! As you can see in the commit history, I was trying out a number of things before Neil gave me the decisive clue. Another thing I tried out was the Gradle Shadow Plugin as I thought I need to create a big-fat-uber-Jar, but then found out that Spring Boot can do this out of the box (unsurprisingly). So I need to tidy up and remove all the obsolete Shadow configs. The repository `jcenter()` was brought in by Shadow, so actually...I would have deleted this along with the rest of the plugin config.